kdc/windc.c

```diff
@@ -82,13 +83,20 @@ generate(krb5_context context, const void *plug, void *plugctx, void *userctx)
 
     if (ft->pac_generate == NULL)
 	return KRB5_PLUGIN_NO_HANDLE;
+
+    if (ft->pac_pk_generate != NULL && uc->pk_reply_key != NULL) {
+	return ft->pac_pk_generate((void *)plug, context,
+				   uc->client, uc->pk_reply_key, uc->pac);
+    }
+
```

Comment:
Can't we just update the signature of `pac_generate` and bump the plugin interface version number? Seems cleaner.